.github/workflows/github.yml

```diff
@@ -229,14 +258,26 @@ jobs:
 
     - name: Configure
       run: |
-        if [[ (${{matrix.preset}} == android-conan-ninja-release) && (${{github.ref}} != 'refs/heads/master') ]]
-        then
-            cmake -DENABLE_CCACHE:BOOL=ON -DANDROID_GRADLE_PROPERTIES="applicationIdSuffix=.daily;signingConfig=dailySigning;applicationLabel=VCMI daily;applicationVariant=daily" --preset ${{ matrix.preset }}
-        elif ${{startsWith(matrix.platform, 'msvc') }}
-        then
+        if [[ ("${{ matrix.preset }}" == "android-conan-ninja-release") && ("${{ github.ref }}" != 'refs/heads/master') ]]; then
+          cmake -DENABLE_CCACHE:BOOL=ON -DANDROID_GRADLE_PROPERTIES="applicationIdSuffix=.daily;signingConfig=dailySigning;applicationLabel=VCMI daily;applicationVariant=daily" --preset ${{ matrix.preset }}
+        elif ${{ startsWith(matrix.platform, 'msvc') }}; then
+          if [[ "${{ matrix.arch }}" == "x86" ]]; then
+            CC="$VCToolsInstallDir/bin/Hostx64/x86/cl.exe"
+            cmake \
+              -D CMAKE_C_COMPILER:FILEPATH="$CC" \
+              -D CMAKE_CXX_COMPILER:FILEPATH="$CC" \
+              --preset ${{ matrix.preset }}
+          elif [[ "${{ matrix.arch }}" == "arm64" ]]; then
+            CC="$VCToolsInstallDir/bin/HostARM64/ARM64/cl.exe"
```

Comment:
Maybe move this path to job parameter for all 3 msvc runs? To avoid this if-else chain?